app/utils/bundle-importer.js

```diff
@@ -91,6 +108,9 @@ YUI.add('bundle-importer', function(Y) {
       @param {String} changesToken The token identifying a bundle change set.
     */
     fetchDryRun: function(bundleYAML, changesToken) {
+      if (bundleYAML) {
+        bundleYAML = this._ensureV4Format(bundleYAML);
```

Comment:
This isn't really a bundleYAML, except in the case where ensureV4Format fails, which I'm assuming is going to be the lesser case.